src/providers/wfs/oapif/qgsoapifitemsrequest.cpp

```diff
@@ -142,6 +143,14 @@ void QgsOapifItemsRequest::processReply()
               mFeatures[i].second = QString::number( id.get<qint64>() );
             }
           }
+          if ( jFeature.is_object() && jFeature.contains( "properties" ) )
+          {
+            const json properties = jFeature["properties"];
```

Comment:
again, if performance critical using jFeature.find( "properties" ) to avoid the double-lookup

Reply:
done